tsconfig.spec.json

```diff
@@ -11,12 +11,13 @@
     ],
     "outDir": "./out-tsc/spec",
     "module": "es2020",
-    "target": "es2020",
+    "target": "ES2022",
     "baseUrl": "tsconfig.spec",
     "types": [
       "jasmine",
       "node"
-    ]
+    ],
+    "useDefineForClassFields": false
```

Comment:
Can you please elaborate on what this one is doing?

Reply:
It's part of auto migration part.

With some research, it seems TS has set this value to true in the 3.7 release. [Ref](https://www.typescriptlang.org/tsconfig#useDefineForClassFields)

And as per [this](https://github.com/microsoft/TypeScript/issues/48814) issue reported, `useDefineForClassFields: true` default setting is potentially breaking the decorators implementations

And since an [issue](https://github.com/angular/components/issues/19221) is open requesting support for **useDefineForClassFields** which still seems to be in discussion

Seems for that reason, angular migrations is setting this to false explicitly not to break any existing project